test/unit/common/schema_dynamic_test.js

```diff
@@ -219,6 +219,7 @@ SchemaDynamicTest.prototype.carpetTesting = function(
 ) {
 	inputs.forEach(input => {
 		it(util.format(description, input.description), done => {
+			// eslint-disable-next-line mocha/no-nested-tests
```

Comment:
How about renaming the variable instead of adding an exception? We have a naming conflict with a Mocha global after all.